app/javascript/convert.js

```diff
@@ -0,0 +1,38 @@
+import {saveAs} from "file-saver";
+
+$(() => {
+  $("form#convert").submit((evt) => {
+    evt.preventDefault();
+    let formData = new FormData();
+    formData.set("file", $("input[type=file")[0].files[0]);
+    formData.set("format", $("select").val());
+    formData.set("historic", $("input[type=checkbox]").is("checked"));
```

Comment:
Looks like the history checkbox is removed? But we're still referencing it here

Reply:
Thought I removed this, will remove the reference to it.